docs/variants/protectli_v1000-series/releases.md

```diff
@@ -0,0 +1,83 @@
+  aggressive CPU power savings, thus not reaching the required performance.
+
+### Fixed
+
+- Auto-boot timeout default value in setup not restored to proper value by F9
+  key
+- Not all Secure Boot settings required a reset
+- Not all Dasharo variables were measured at boot
+
+### Binaries
+
+[protectli_v1210_v0.9.2.rom][protectli_v1210_v0.9.2.rom_file]{.md-button}
+[sha256][protectli_v1210_v0.9.2.rom_hash]{.md-button}
+[sha256.sig][protectli_v1210_v0.9.2.rom_sig]{.md-button}
+
+[protectli_v1211_v0.9.2.rom][protectli_v1211_v0.9.2.rom_file]{.md-button}
+[sha256][protectli_v1211_v0.9.2.rom_hash]{.md-button}
+[sha256.sig][protectli_v1211_v0.9.2.rom_sig]{.md-button}
+
+To verify binary integrity with hash and signature please follow the
+instructions in [Dasharo release signature
+verification](/guides/signature-verification) using [this
+key](https://github.com/3mdeb/3mdeb-secpack/raw/master/customer-keys/protectli/release-keys/dasharo-release-0.9.x-for-protectli-signing-key.asc)
+
+### SBOM (Software Bill of Materials)
+
+- [coreboot based on 4.21 revision 147e688e](https://github.com/Dasharo/coreboot/tree/147e688e)
+- [Dasharo EDKII fork based on edk2-stable202002 revision d130aece](https://github.com/Dasharo/edk2/tree/d130aece)
+- [iPXE based on 2023.12 revision 838611b3](https://github.com/Dasharo/ipxe/tree/838611b3)
+- [Intel Management Engine based on v13.50.11.1304 revision 9712e97a](https://github.com/Dasharo/dasharo-blobs/blob/9712e97a/protectli/vault_jsl/V1210/)
+- [Intel Flash Descriptor based on v1.0 revision 9712e97a](https://github.com/Dasharo/dasharo-blobs/blob/9712e97a/protectli/vault_jsl/V1210/)
+- [Intel Firmware Support Package based on JSL 2021/08/23 v2115 revision 9712e97a](https://github.com/Dasharo/dasharo-blobs/blob/9712e97a/protectli/vault_jsl/JasperLakeFspBinPkg)
+- [Intel microcode based on JSL A0 0x24000026 revision microcode-20230808](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/tree/microcode-20230808/intel-ucode/06-9c-00)
+
+[newsletter]: https://newsletter.3mdeb.com/subscription/n2EpSxtqL
+[protectli_v1210_v0.9.2.rom_file]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1210_v0.9.2.rom
+[protectli_v1210_v0.9.2.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1210_v0.9.2.rom.sha256
+[protectli_v1210_v0.9.2.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1210_v0.9.2.rom.sha256.sig
+[protectli_v1211_v0.9.2.rom_file]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1211_v0.9.2.rom
+[protectli_v1211_v0.9.2.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1211_v0.9.2.rom.sha256
+[protectli_v1211_v0.9.2.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/protectli_vault_jsl/v0.9.2/protectli_v1211_v0.9.2.rom.sha256.sig
```

Comment:
This file is part of the release, so if this is not supposed to be a PR with release content, it should be removed.